frontend/src/components/pages/UserProfilePage.tsx

```diff
@@ -84,6 +84,11 @@ const UserProfilePage = () => {
     StoryAssignStage.INITIAL,
   );
 
+  const [resume, setResume] = useState<File | null>(null);
+
+  // TODO: Send resume to backend when save changes button is implemented
+  console.log(resume);
```

Comment:
Is the TODO comment ok?

Reply:
I added `console.log(resume);` to avoid the `no-unused-vars` linter error (since the code using `resume` isn't implemented yet). Is there a way to avoid the error?

Reply:
yesss add `  /* eslint-disable-next-line @typescript-eslint/no-unused-vars */` right before you declare the useState line